brevbaker-dsl/src/main/kotlin/no/nav/pensjon/brev/template/LetterTemplate.kt

```diff
@@ -78,22 +96,42 @@ sealed class Expression<out Out> : StableHash {
                 }
 
             override fun stableHashCode() = hashCode()
+
+            override fun equals(other: Any?): Boolean {
+                if (other !is Assigned<*>) return false
+                return id == other.id
+            }
+            override fun hashCode() = Objects.hash(id)
         }
     }
 
-    data class UnaryInvoke<In, Out>(
+    // TODO: Vurder intern konstruktør
```

Comment:
Både UnaryInvoke og BinaryInvoke må kunne instantieres utenfor dsl-modulen, siden vi ønsker å tilby muligheten for å lage egne operations. Så om konstruktøren er intern, så må vi tilby en annen måte å instantiere på.